pkg/scheduler/scheduler.go

```diff
@@ -110,8 +110,7 @@ func (s *Scheduler) schedule(ctx context.Context) {
 		c := snapshot.ClusterQueues[e.ClusterQueue]
 		if len(e.borrows) > 0 && c.Cohort != nil && usedCohorts.Has(c.Cohort.Name) {
 			e.admissible = false
-			// TODO(): we shouldn't emit an event or update the workload condition in this
-			// case, just re-queue directly.
+			e.skipped = true
 			e.noAdmissionReason = "cohort used in this cycle"
```

Comment:
inadmissibleReason

Reply:
I think it can be renamed in other pr https://github.com/kubernetes-sigs/kueue/pull/185